### ISSUE TITLE
I can't find the InstallCLI plugin in my Xcode

### ISSUE THREAD

**Issue by Santiago0412** [apollographql/apollo-ios#2734]
### Summary

From the document it says I can install the CLI by doing this:
<img width="681" alt="image" src="https://user-images.githubusercontent.com/10152008/207851976-32720913-cbfe-4237-8333-e20e8323ddde.png">

but in my Xcode it looks like this:
<img width="236" alt="image" src="https://user-images.githubusercontent.com/10152008/207852098-74139ee6-884a-42ec-ad32-63fc55b279d7.png">

Xcode: 14.2
System:13.0.1 (22A400)
Machine: Mac air m1

### Version

1.0.0

### Steps to reproduce the behavior

- Install the package;
- Right click Apollo package;

**Comment by calvincestari** [comment]
Hi @Santiago0412 👋 - see [this comment](https://github.com/apollographql/apollo-ios/pull/2683#issuecomment-1352600855). I've got a PR up to change the screenshot already.

**Comment by Santiago0412** [comment]
@calvincestari Can you also update this link: seems it not working 
<img width="641" alt="image" src="https://user-images.githubusercontent.com/10152008/208016200-272fb145-0c89-4173-bc01-43f7afe14e83.png">

**Comment by calvincestari** [comment]
Yes, we'll get that fixed - thanks for letting me know.

**Comment by calvincestari** [comment]
Hi @teovillanueva - 👋🏻 

tl;dr - you'll need to update to Xcode 14 for this to work on an Xcode project.

Xcode can open two project types:
1. Swift package - this is when your project has a `Package.swift` file
2. Xcode project - this is when your project has an `xcodeproj`/`xcworkspace` file.

Xcode 13.3 ([release notes](https://developer.apple.com/documentation/xcode-release-notes/xcode-13_3-release-notes)) only supports plugins for Swift packages, but Xcode 14 ([release notes](https://developer.apple.com/documentation/xcode-release-notes/xcode-14-release-notes)) supports plugins for both Swift packages _and_ Xcode projects. [This](https://blog.eidinger.info/xcode-integration-of-swift-package-plugins-in-xcode-14) is a good article explaining the differences.

**Comment by nimeshneema** [comment]
@calvincestari I am running into this issue with just updated version of Xcode 14.3. Not able to see the InstallCLI command in the context menu.

**Comment by calvincestari** [comment]
https://github.com/apollographql/apollo-ios/issues/2919#issuecomment-1492481609

**Comment by BharatBapodara** [comment]
Hi @calvincestari,
I can not see Install CLI on M1 Mac can you please check screenshots (it is working on intel one)

<img width="1352" alt="Screenshot 2023-05-31 at 09 09 39" src="https://github.com/apollographql/apollo-ios/assets/54742454/e73acf9b-2020-4b08-89c5-eda0177ffd4f">
<img width="1352" alt="Screenshot 2023-05-31 at 09 09 44" src="https://github.com/apollographql/apollo-ios/assets/54742454/bb6b03c2-4157-4edb-891b-fa1983d21a58">

**Comment by BobaFetters** [comment]
@BharatBapodara What version of Xcode are you running? There is a known issue with Xcode 14.3 where SPM Plugins don't work correctly, which has been resolved in Xcode 14.3.1, #2919 has some discussion around this.
